### DISCUSSION TITLE
Do logs in ARC runner pods include logs in GitHub Actions workflow run?

### DISCUSSION THREAD

**Discussion by YokojimaSkewers** [actions/actions-runner-controller#2054] [Questions]
I would like to know the answer of the question in title.
If GitHub Secrets are used in GitHub Actions, they are redacted in logs in GitHub Actions workflow run as we can see at the GitHub webpage.
But I'm not sure whether logs in ARC runner pods also have the same logs and secrets, which might not be redacted anymore.
I appreciate in advance.

**Comment by acutillas-gs** [comment]
I don't see the logs that are printed in github in either the runner or the controller logs
